Testing/VelaptorTests/OpenGL/GLWindowTests.cs

```diff
@@ -681,26 +684,29 @@ public void WindowState_WhenGettingInvalidValue_ThrowsException()
         var act = () => _ = sut.WindowState;
 
         // Assert
-        act.Should().Throw<EnumOutOfRangeException<WindowState>>()
+        act.Should()
+            .Throw<InvalidEnumArgumentException>()
             .WithMessage(expected);
     }
 
     [Fact]
     public void WindowState_WhenSettingInvalidValue_ThrowsException()
     {
         // Arrange
-        var expected = $"The value of the enum '{nameof(StateOfWindow)}' used in the class '{nameof(GLWindow)}' and";
-        expected += " method 'SetupOtherPropCaches' is invalid and out of range.";
+        var invalidValue = 1234;
```

Comment:
Please refactor this to a constant

```suggestion
        const invalidValue = 1234;
```